src/meshlink.c

```diff
@@ -2271,19 +1958,10 @@ bool meshlink_set_port(meshlink_handle_t *mesh, int port) {
 	}
 
 	close_network_connections(mesh);
-	exit_configuration(&mesh->config);
-
-	char portstr[10];
-	snprintf(portstr, sizeof(portstr), "%d", port);
-	portstr[sizeof(portstr) - 1] = 0;
-
-	modify_config_file(mesh, mesh->name, "Port", portstr, true);
 
-	init_configuration(&mesh->config);
+	// TODO: write meshlink.conf again
```

Comment:
Need to set `mesh->myport` with the new port value and write to the meshlink main config file